### DISCUSSION TITLE
Link to the MyAnimeList/Anilist Manga that was Tracked

### DISCUSSION THREAD

**Discussion by Kinasho** [xgi/houdoku#142] [Feature Requests]
I would like for there to be a link to the MyAnimeList/Anilist page for a manga that I have tracked. 
![image](https://user-images.githubusercontent.com/106453771/170845749-5cf332f1-7f1f-41fc-9ef9-3dd8dcb44c6a.png)
For example, there can be an Anilist link to One Piece here

Also, maybe as a secondary option, Mangadex has links to the MyAnimeList/Anilist manga that they have, so perhaps links can be provided for the manga pages on the "Add Series to Library" page. 
![image](https://user-images.githubusercontent.com/106453771/170845805-cb40a01a-241e-42b9-be2b-04fdefc768f9.png)
Also, perhaps this can be used in tandem with the tracking features, whereby users who track using MyAnimeList would show a MyAnimeList link, and Anilist for Anilist tracking users.

**Comment by xgi** [comment]
Adding the AniList/MAL link for series that the user has linked is definitely reasonable, I can add that.

For your second point, I think we could maybe add a series field like "External Links" which could pull the AniList/MAL link from the extension, but I think MangaDex will probably be the only extension that ever has those, so I'm not sure it would be broadly useful.

**Reply by Kinasho** [reply]
Seems reasonable. Thanks for replying!
